docker/services/simplified_crontab

```diff
@@ -107,8 +107,14 @@ HOME=/var/www/circulation
 #
 0 6 * * * root core/bin/run odl_import_monitor >> /var/log/cron.log 2>&1
 0 */8 * * * root core/bin/run odl_hold_reaper >> /var/log/cron.log 2>&1
+0 */15 * * * root core/bin/run odl_reaper >> /var/log/cron.log 2>&1
```

Comment:
Did you mean every 15 minutes ( */15 * * * * ) here?

Reply:
🙏🏻